README.md

```diff
@@ -217,7 +211,7 @@ in a local directory:
       Or install the latest version from
       [python.org](https://www.python.org/downloads/windows) for Windows.
 
-4. Windows only: Download [adb](https://developer.android.com/tools/releases/platform-tools)
+3. Windows only: Download [adb](https://developer.android.com/tools/releases/platform-tools)
```

Comment:
Should we create a pre-requisite/env setup section, and move all relevant info like the python setup there?

Technically the linux host needs to install adb as well